pipelines/GERPonHAL/gerp_on_hal.nf

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env nextflow
 
+genomes_file = file(params.genomes_file)
+tree_file = file(params.tree_file)
```

Comment:
Would the Newick tree in the HAL file be sufficient for GERP? If yes, I'd suggest adding a process to do this. The command to extract the tree goes something like this:
```
halStats --tree input.hal > output.nh
```

Reply:
CACTUS needs a tree file as input, and I would expect that one and the one generated by `halStats` to be the same, so I think this can be left as it is (but good to know about this tool :+1:).